lib/ansible/cli/doc.py

```diff
@@ -71,11 +72,36 @@ class DocCLI(CLI):
     # default ignore list for detailed views
     IGNORE = ('module', 'docuri', 'version_added', 'short_description', 'now_date', 'plainexamples', 'returndocs', 'collection')
 
+    # Warning: If you add more elements here, you also need to add it to the docsite build (in the
+    # ansible-community/antsibull repo)
+    _ITALIC = re.compile(r"\bI\(([^)]+)\)")
+    _BOLD = re.compile(r"\bB\(([^)]+)\)")
+    _MODULE = re.compile(r"\bM\(([^)]+)\)")
+    _LINK = re.compile(r"\bL\(([^)]+), *([^)]+)\)")
+    _URL = re.compile(r"\bU\(([^)]+)\)")
+    _REF = re.compile(r"\bR\(([^)]+), *([^)]+)\)")
+    _CONST = re.compile(r"\bC\(([^)]+)\)")
+    _RULER = re.compile(r"\bHORIZONTALLINE\b")
+
     def __init__(self, args):
 
         super(DocCLI, self).__init__(args)
         self.plugin_list = set()
 
+    @classmethod
+    def tty_ify(cls, text):
+
+        t = cls._ITALIC.sub(r"`\1'", text)    # I(word) => `word'
+        t = cls._BOLD.sub(r"*\1*", t)         # B(word) => *word*
```

Comment:
This may not matter because something else may handle the RST --> HTML conversion, but for in RST, `**word**` is the syntax for bold while `*word*` is the syntax for italic. If this is only for the display of `ansible-doc`, this should be fine. But it'd be surprising if `B(word)` ended up being italic in the final HTML.

Reply:
This isn't rst.

Reply:
rst and html (for building the web docs) are handled by rst_ify and html_ify from this code: https://github.com/ansible-community/antsibull/blob/main/antsibull/jinja2/filters.py

The code in this PR is only for ansible-doc to output to the screen.

This split is why I added the note that the code in both places needs to be changed if new macros are added.

Reply:
Ah, ok. I figured they were separate and this was only for `ansible-doc` output. I just wanted to make sure.

Reply:
Fun times with cross project relationships. 😁